wordpress-mobile/WordPress-iOS
# Sharing: Adds new icons for new services. Fixes glitch in google's icon.

Fixes #5136

To test:
View the blog details for a wpcom site.
View sharing settings.
View sharing buttons.
Confirm that the icons for Google, WhatsApp, Skype, and Telegram are properly displayed and not using the default share icon.

Needs review: @kurzee

Same as #5197 but targeting the correct branch.
